spec/models/applicant_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
Can this be moved to `.rspec` file?

Reply:
I like this. Should I do it for all the `*_spec.rb` files?